### PR TITLE
Implemented Singleton Log Class

### PR DESCRIPTION
**Pull request by Regis18**
I send the Log class in order to have a unique object of logs

### CODE REVIEW

File: src/main/java/com/jala/utils/Logs.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+     */
+    public static Logs getInstance(){
+        if (instance == null) {
+            instance = new Logs();
+            PropertyConfigurator.configure("../ToolSearch/src/main/java/com/jala/resources/log4j.properties");
+            log.setLevel(Level.ALL);
+        }
+        return instance;
+    }
+
```
Comment (jpsandovaln) [inline src/main/java/com/jala/utils/Logs.java:48]: Remove extra spaces.

File: src/main/java/com/jala/utils/Logs.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
+package com.jala.utils;
```
Comment (jpsandovaln) [inline src/main/java/com/jala/utils/Logs.java:13]: Add copyright

Comment (Regis18) [reply src/main/java/com/jala/utils/Logs.java:13]: I'll do it

File: src/main/java/com/jala/utils/Logs.java (hunk 1 above)
Comment (Regis18) [reply src/main/java/com/jala/utils/Logs.java:48]: ok

**Pull request closed without merging**